apps/admin/backend/src/perf.test.ts

```diff
@@ -0,0 +1,70 @@
+import { computeFullElectionTally } from '@votingworks/utils';
+import { join } from 'path';
+import { electionMinimalExhaustiveSampleDefinition } from '@votingworks/fixtures';
+import { assert } from '@votingworks/basics';
+import { CastVoteRecord } from '@votingworks/types';
+import {
+  buildTestEnvironment,
+  configureMachine,
+  mockElectionManagerAuth,
+} from '../test/app';
+import { getBackupPath, takeBackup } from '../test/backups';
+import { getPerformanceTimer } from '../test/timer';
+
+// depending on the test conditions, may need to increase this
+jest.setTimeout(600000);
+
+const electionDefinition = electionMinimalExhaustiveSampleDefinition;
+
+test.skip('load CVR files', async () => {
```

Comment:
nit: Is it worth moving this file elsewhere, like `admin/backend/perf` or `admin/backend/scripts` and having a manual way to run it (maybe a separate `pnpm test:perf` command)?
Might not be clear at first glance why these are being skipped (but +1 to committing the perf test code for reuse)

Reply:
yeah, I think that would be better. I looked into it briefly but wasn't sure how to set up `jest` to run that way. it must be possible though, will make it work.

Reply:
Sounds good - can definitely be a followup later down the line, though. I think this is fine for now, to get things moving

Reply:
I figured out how to do the `jest` regex's the right way, and it's a big improvement! Thanks for the prod, 6b703d5.

Reply:
Oh nice! 🙌